### ISSUE TITLE
Wrong check for python-config

### ISSUE THREAD

**Issue by jozefkucia** [baldurk/renderdoc#647]
A check for `python-config` in `qrenderdoc/CMakeLists.txt` doesn't allow me to build renderdoc on `Gentoo`:
```
-- Found PythonLibs: /usr/lib64/libpython3.4.so (found suitable version "3.4.1", minimum required is "3") 
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.4.1", minimum required is "3") 
CMake Error at qrenderdoc/CMakeLists.txt:57 (message):
  We require python3-config to build swig, please install the python dev
  package for your system.
```

Removing the check lets the build complete.

On my system python-config for 3.4.1 is available at `/usr/bin/python3.4-config` and `/usr/bin/python-config-3.4`

**Comment by baldurk** [comment]
OK, I thought the python3 executable was always `python3` (and so `python3-config` follows). I'll extend the check to look for `python${MAJOR}.${MINOR}-config` as well, that seems to be what SWIG does.

**Comment by baldurk** [comment]
I think that commit should sort it out. I tested building on a Gentoo VM and it worked OK, but on my VM I had a `python3-config` in my path at `/usr/bin` so let me know if this still doesn't work for you.

**Comment by jozefkucia** [comment]
https://github.com/baldurk/renderdoc/pull/648 fixes the problem for me

**Comment by baldurk** [comment]
Closing with merged #648.
